contracts/Staking.sol

```diff
@@ -273,6 +275,7 @@ contract Staking is IStaking, Initializable, OwnableUpgradeable {
             totalStakingAmount[_indexer].valueAt = _amount;
             delegation[_source][_indexer].valueAfter = _amount;
             totalStakingAmount[_indexer].valueAfter = _amount;
+            IStakingAllocation(settings.getContractAddress(SQContracts.StakingAllocation)).update(_indexer, totalStakingAmount[_indexer].valueAt);
```

Comment:
I think probably the best place to add this callback is in RewardsStaking.applyStakeChange()